libs/escrow-hooks/src/index.ts

```diff
@@ -1,14 +1,5 @@
-export { default as useDebounce } from './useDebounce';
-export { default as useDeposit } from './useDeposit';
-export { default as useEscrowZap } from './useEscrowZap';
+export { default as useDetailsPin } from './useDetailsPin';
```

Comment:
`useDetailsPin` is also available in `@smartinvoicexyz/hookes`
(https://github.com/SmartInvoiceXYZ/smart-invoice/blob/develop/packages/hooks/src/useDetailsPin.ts)
Could we move to use that in DM directly and remove useDetailsPin from DM?